spec/mailers/subscription_mailer_spec.rb

```diff
@@ -252,7 +252,6 @@
       end
 
       it "renders the shop's logo" do
-        shop.update!(logo: fixture_file_upload("logo.png", "image/png"))
```

Comment:
This is great practice in development but you don't need this as separate commit in the PR. In such a small PR, it can still be nice to demonstrate the test driven development as you are doing here but then I would add `pending "new factory trait for a logo` so that the spec is expected to fail and the build is green.

I like green builds with every commit if possible because it makes running `git bisect` easier to find a regression.

Reply:
Ah, yes - that makes sense. Instead of the red-build to have the pending example. Will do so next time, thanks :pray: